src/rules/handleCallbackErrRule.ts

```diff
@@ -67,10 +80,21 @@ class ErrCallbackHandlerWalker extends Lint.RuleWalker {
   private stack: IFunctionScope[] = [];
   private errorCheck: (name: string) => boolean;
   private firstParameterName: (node: ts.FunctionLikeDeclaration) => string | undefined;
+  private allowProperties: boolean = true;
 
   constructor(sourceFile: ts.SourceFile, options: Lint.IOptions) {
     super(sourceFile, options);
-    const errorArgument = options.ruleArguments[0] || 'err';
+    const opt = this.getOptions();
+    let errorArgument = 'err';
+    let optObj = opt[0];
+    if (typeof opt[0] === 'string') {
+      errorArgument = opt[0];
+      optObj = opt[1];
+    }
+
+    if (optObj) {
+      this.allowProperties = !!optObj.allowProperties;
```

Comment:
Instead of doing this check, I'd recommend using `this.allowProperties = optObj.allowProperties !== false` as an empty object would equal `false` right now which is inconsistent with non-existence.